src/components/HomePageNavigation/HomePageNavigation.jsx

```diff
@@ -11,9 +17,16 @@ const HomePageNavigation = () => {
       </Link>
       <ul className="home-links flex-row flex-align-center">
         <li>
-          <NavLink className={`home-link link-none `} to="/login">
-            LOGIN
-          </NavLink>
+          {isLoggedIn ?
+            <button className={`hero-btn hero-btn-primary py-2 px-3`} onClick={() => {
+              logoutService(navigate, setAuthState);
+            }} >
+              LOGOUT
+            </button> :
+            <NavLink className={`home-link link-none`} to="/login">
```

Comment:
Remove deafult active class styling as it shows underline decoration

Reply:
Didn't understand what you are trying to convey!!Can you show it through a snapshot?